### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by MatricksDeCoder**
Current status of project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,19 +22,10 @@ To have the interface default to a different network when a wallet is not connec
 2. Change `REACT_APP_NETWORK_ID` to `"{YOUR_NETWORK_ID}"`
 3. Change `REACT_APP_NETWORK_URL` to e.g. `"https://{YOUR_NETWORK_ID}.infura.io/v3/{YOUR_INFURA_KEY}"` 
 
-Note that the interface only works on testnets where both 
-[Uniswap V2](https://uniswap.org/docs/v2/smart-contracts/factory/) and 
-[multicall](https://github.com/makerdao/multicall) are deployed.
-The interface will not work on other networks.
+Note that the interface currently only works where BTRUST and BTRUST Governane contracts are deployed on Ropstein networ
```
Comment (musnit) [inline README.md:25]:
```suggestion
Note that the interface currently only works where BTRUST and BTRUST Governance contracts are deployed on Ropsten network
```

File: src/constants/index.ts [hunk 2]
```diff
@@ -1,47 +1,33 @@
-import { ChainId, JSBI, Percent, Token, WETH } from '@uniswap/sdk'
+import { ChainId, JSBI, Percent, Token } from '@uniswap/sdk'
 import { AbstractConnector } from '@web3-react/abstract-connector'
 
 import { fortmatic, injected, portis, walletconnect, walletlink } from '../connectors'
 
-export const ROUTER_ADDRESS = '0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D'
+//export const ROUTER_ADDRESS = '0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D'
```
Comment (musnit) [inline src/constants/index.ts:6]: is this used anywhere else? can just delete it if not?

File: src/pages/Vote/index.tsx [hunk 3]
```diff
@@ -152,14 +153,13 @@ export default function Vote() {
               </RowBetween>
               <ExternalLink
                 style={{ color: 'white', textDecoration: 'underline' }}
-                href="https://uniswap.org/blog/uni"
+                href="http://wwww.google.com"
```
Comment (musnit) [inline src/pages/Vote/index.tsx:162]: can just link to the main braintrust website for now until we have our blog/docs up

**Review by musnit — APPROVED** [review]

**Comment by MatricksDeCoder** [comment]
Merge main with staging

**Pull request merged**